cb-sol-cli/cmd/bridge.js

```diff
@@ -0,0 +1,101 @@
+const ethers = require('ethers');
+const constants = require('../constants');
+
+const {Command} = require('commander');
+const {setupParentArgs, getFunctionBytes, waitForTx, log} = require("./utils")
+
+const EMPTY_SIG = "0x00000000"
+
+const registerResourceCmd = new Command("register-resource")
+    .description("Register a resource ID with a contract address for a handler")
+    .option('--bridge <address>', 'Bridge contract address', constants.BRIDGE_ADDRESS)
+    .option('--handler <address>', 'Handler address', constants.ERC20_HANDLER_ADDRESS)
+    .option('--targetContract <address>', `Contract address to be registered`, constants.ERC20_ADDRESS)
+    .option('--resourceId <address>', `Resource ID to be registered`, constants.ERC20_RESOURCEID)
+    .action(async function (args) {
+        await setupParentArgs(args, args.parent.parent)
+
+        const bridgeInstance = new ethers.Contract(args.bridge, constants.ContractABIs.Bridge.abi, args.wallet);
+        log(args,`Registering contract ${args.targetContract} with resource ID ${args.resourceId} on handler ${args.handler}`);
+        const tx = await bridgeInstance.adminSetResource(args.handler, args.resourceId, args.targetContract, { gasPrice: args.gasPrice, gasLimit: args.gasLimit});
+        await waitForTx(args.provider, tx.hash)
+    })
+
+const registerGenericResourceCmd = new Command("register-generic-resource")
+    .description("Register a resource ID with a generic handler")
+    .option('--bridge <address>', 'Bridge contract address', constants.BRIDGE_ADDRESS)
+    .option('--handler <address>', 'Handler contract address', constants.GENERIC_HANDLER_ADDRESS)
+    .option('--targetContract <address>', `Contract address to be registered`, constants.CENTRIFUGE_ASSET_STORE_ADDRESS)
+    .option('--resourceId <address>', `ResourceID to be registered`, constants.GENERIC_RESOURCEID)
+    .option('--deposit <string>', "Function signature of the deposit functions", EMPTY_SIG)
+    .option('--execute <string>', "Function signature of the proposal execution function", EMPTY_SIG)
+    .option('--hash', "Treat signature inputs as function signature strings, hash and take the first 4 bytes", false)
```

Comment:
```suggestion
    .option('--hash', "Treat signature inputs as function prototype strings, hash and take the first 4 bytes", false)
```
sry, just a small semantic thing: in solidity docs the function signature/selector is defined as `The first 4 bytes of the Keccak256 hash of the function’s prototype.`